README.md

```diff
@@ -25,7 +29,8 @@ In this lab, we're going to practice finding elements in the DOM. To do so, we'r
 </body>
 ```
 
-If we called `document.querySelector('div')`, the method would return the first `div` (whose `.innerHTML` is "Hello!").
+If we called `document.querySelector('div')`, the method would return the first
+`div` (whose `.innerHTML` is "Hello!").
```

Comment:
This feels like the .innerHTML is related to the selection's result. 

(whose _content_ is "Hello!"). 

?